history-network.md

```diff
@@ -0,0 +1,300 @@
+# Chain History Storage Network
+
+This document is a preliminary specification for a networking protocol that supports on-demand availability of Ethereum chain history data.
+
+## Overview
+
+Chain history data consists of historical block headers and block bodies, where a block body consists of transactions and transaction receipts.
+
+The data stored in the chain history storage network will support the following [eth](https://github.com/ethereum/devp2p/blob/master/caps/eth.md) protocol requests:
```

Comment:
I think this statement might be worth re-working since the network doesn't actually support those requests, but rather, 
it support retrieving the underlying data returned by those requests.

For example:

- `GetBlockHeaders` returns up to 384 headers, with various parameters for querying by block numbers and hashes, as well as asking for every Nth header across a range of headers.
- State network will only support retrieval of individual headers by their hash

Reply:
Right. That is what I meant when I wrote that the data stored here would support those requests. I do agree that it's not clear that the network itself cannot actually serve those requests though, so I will re-word it.